iOSDeviceManager/IDMVersionDefines.h

```diff
@@ -1,11 +1,6 @@
-
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
 /*
  Do Not Manually Edit This File
 */
-
-#define IDM_GIT_SHORT_REVISION @"94c9a5f"
-#define IDM_GIT_BRANCH @"update-cli"
+#define IDM_GIT_SHORT_REVISION @"679df5a"
```

Comment:
Should it be removed?

Reply:
Actually, I don't know. It was set automatically during the application building.